tests/contrib/hooks/test_bigquery_hook.py

```diff
@@ -337,8 +337,28 @@ def run_with_config(config):
         mocked_rwc.assert_called_once()
 
 
-class TestTimePartitioningInRunJob(unittest.TestCase):
+class TestDatasetsOperations(unittest.TestCase):
+    @mock.patch.object(hook.BigQueryBaseCursor, 'run_with_configuration')
+    def test_create_empty_dataset(self, run_with_configuration):
```

Comment:
The `test_create_empty_dataset` name of test doesn't match with the tests conducted.